includes/class-genesis-simple-share-preview.php

```diff
@@ -163,6 +166,7 @@ public function get_icon_output( $location, $icons = array() ) {
 									  urlCurl: '%s',
 									  enableHover: false,
 									  enableTracking: true,
+										%s
```

Comment:
@marksabbath Looks like the alignment may be off here. Other lines also include tabs and spaces.

Reply:
It should be fixed by the last commit @dreamwhisper :)